x/fungible/keeper/zrc20_cosmos_coins_mapping.go

```diff
@@ -25,7 +25,7 @@ func (k Keeper) LockZRC20(
 ) error {
 	// owner is the EOA owner of the ZRC20 tokens.
 	// locker is the address that will lock the ZRC20 tokens, i.e: bank precompile.
-	if err := k.CheckZRC20Allowance(ctx, zrc20ABI, owner, locker, zrc20Address, amount); err != nil {
+	if err := k.CheckZRC20Allowance(ctx, zrc20ABI, owner, spender, zrc20Address, amount); err != nil {
```

Comment:
Can we modify the comment to reflect the change

Reply:
Applied in 06d0c5d5a8a2d7ce54f84a666c7bef7784963ccf